src/stacks-hierarchy/Root_PurchaseConfirmationScreen/components/SelectPaymentMethodSheet.tsx

```diff
@@ -92,30 +87,28 @@ export const SelectPaymentMethodSheet: React.FC<Props> = ({
   const [selectedOption, setSelectedOption] = useState<
     PaymentMethod | undefined
   >(getSelectedPaymentMethod(paymentTypes, previousPaymentMethod));
-  const [remoteOptions, setRemoteOptions] = useState(remotePaymentOptions);
   const styles = useStyles();
 
-  async function getRecurringPaymentOptions(): Promise<
-    Array<SavedPaymentOption>
-  > {
+  function getRecurringPaymentOptions(): SavedPaymentOption[] {
     if (authenticationType !== 'phone') return [];
-    const recurringOptions: Array<SavedPaymentOption> = (
-      await listRecurringPayments()
-    ).map((option) => {
-      return {
-        savedType: 'recurring',
-        paymentType: option.payment_type,
-        recurringCard: {
-          id: option.id,
-          masked_pan: option.masked_pan,
-          expires_at: option.expires_at,
-          payment_type: option.payment_type,
-        },
-      };
-    });
-    return [...recurringOptions.reverse()];
+    const recurringOptions: Array<SavedPaymentOption> | undefined =
```

Comment:
```suggestion
    const recurringOptions: Array< RecurringPaymentOption > | undefined =
```
Since it never returns `DefaultPaymentOption`, it is more precise to return `RecurringPaymentOption`.

Reply:
It will all be `PaymentOption` after https://github.com/AtB-AS/mittatb-app/compare/rosvik/update-recurring-payments...rosvik/refactor-payment-types?expand=1